### PR TITLE
CTCP-2330: Nav for Representative section and clean up

### PR DESCRIPTION
**Pull request by kevincoatsworth**

### CODE REVIEW

File: app/api/submission/Representative.scala [hunk 1]
```diff
@@ -26,8 +26,10 @@ object Representative {
       r =>
         RepresentativeType05(
           r.eori.value,
-          r.capacity.code,
-          Some(ContactPersonType05(r.name, r.phone, None))
+          "direct",
```
Comment (jackheslop96) [inline app/api/submission/Representative.scala:29]: I think the code for direct is `"2"` judging by this - https://github.com/hmrc/manage-transit-movements-departure-frontend/blob/11dad387b93264be6ccf4bf41e9ffb5020623f36/app/models/traderDetails/representative/RepresentativeCapacity.scala#L28

Comment (kevincoatsworth) [reply app/api/submission/Representative.scala]: Yea you're correct, it should be using the code that was associated with the object - pushed that

File: app/api/submission/Representative.scala [hunk 2]
```diff
@@ -26,8 +26,10 @@ object Representative {
       r =>
         RepresentativeType05(
           r.eori.value,
-          r.capacity.code,
-          Some(ContactPersonType05(r.name, r.phone, None))
+          "2",
```
Comment (ajchilts) [inline app/api/submission/Representative.scala:29]: Will this always be 2?

Comment (kevincoatsworth) [reply app/api/submission/Representative.scala:29]: I believe so from looking at ticket CTCP-2329

**Review by ajchilts — APPROVED** [review]

**Pull request merged**